src/aiida/tools/archive/create.py

```diff
@@ -280,94 +288,130 @@ def querybuilder():
 
     EXPORT_LOGGER.report(f'Creating archive with:\n{tabulate(count_summary)}')
 
+    # Handle temporary directory configuration
+    tmp_prefix = '.aiida-export-'
```

Comment:
Why define this variable, instead of just passing its value in the `TemporaryDirectory` constructor?